### PR TITLE
updated default symbols server url

### PR DESCRIPTION
**Pull request by xavierdecoster**
for NuGet.Core 2.10
https://github.com/NuGet/Home/issues/1583

### CODE REVIEW

**Comment by xavierdecoster** [comment]
cc @MeniZalzman @emgarten

**Comment by emgarten** [comment]
Is going from http to https here going to be a problem for anyone?

:shipit:

**Comment by xavierdecoster** [comment]
@emgarten don't think so, server certificate for that endpoint looks OK to me

**Comment by xavierdecoster** [comment]
@MeniZalzman @yishaigalatzer you OK with this change?

**Comment by yishaigalatzer** [comment]
Needs a doc and will need to on the release announcement. @csharpfritz

Other than that looks good

**Comment by xavierdecoster** [comment]
This is a related doc PR that's already live: https://github.com/NuGet/NuGetDocs/pull/386

**Comment by yishaigalatzer** [comment]
Please unmerge it, docs should only go in once the product goes live. It should also indicate that push is different across different versions of nuget.exe

Sent from my Windows Phone

**Comment by xavierdecoster** [comment]
Reverted the change in the docs site and reopened a new PR with those changes: https://github.com/NuGet/NuGetDocs/pull/390

Note: the intent of the docs PR was to have people start using the new, faster symbolsource endpoints instead of the old one that has issues.
Not sure whether that docs change was depending on actually changing the default in the code, but @TripleEmcoder might clarify.

**Comment by TripleEmcoder** [comment]
The change in code is independent from the docs. The old URL redirects to the new one internally, so there is no loss of data or functionality. With a change in the docs, new users could start using the new endpoint ASAP, but it might be confusing for them to see one URL in the docs and a different one when running from the console. Your call, both options are fine as long as we get both changes deployed in a reasonable timeframe.

**Comment by TripleEmcoder** [comment]
As for SSL we could do HTTP, there is no server side redirect forcing HTTPS. Users could also override that wither way with -source. I believe SSL might be a better default, as all of the net is going that way.

**Comment by MeniZalzman** [comment]
:shipit:

File: test/CommandLine.Test/PushCommandTest.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ private static IPackageSourceProvider CreateSourceProvider()
         }
 
         [Theory]
-        [InlineData(new object[] { @"X:\test\foobar.symbols.nupkg", @"http://nuget.gw.symbolsource.org/Public/NuGet" })]
+        [InlineData(new object[] { @"X:\test\foobar.symbols.nupkg", @"https://nuget.smbsrc.net/" })]
```
Comment (yishaigalatzer) [inline test/CommandLine.Test/PushCommandTest.cs]: while you are at it, get rid of foobar

Comment (xavierdecoster) [inline test/CommandLine.Test/PushCommandTest.cs]: removed all occurrences of `foo`, `bar` and `foobar` in PushCommandTest.cs in https://github.com/NuGet/NuGet2/commit/fa862eff17059b02e48cab7845b0d94101263192

**Comment by xavierdecoster** [comment]
@MeniZalzman all yours :)

**Comment by emgarten** [comment]
@xavierdecoster this didn't go into nuget.exe 3.3.0?

**Comment by emgarten** [comment]
At this point I think this PR should be closed. This is just to change a constant that NuGet.Client uses from NuGet.Core. Just make the change in NuGet.Client instead of referencing it from here.

**Comment by xavierdecoster** [comment]
@emgarten it did make it to nuget.exe 3.3 https://github.com/NuGet/NuGet.Client/pull/46

This was a left-over PR on the old NuGet.Core.dll. If there's no need to update NuGet.Core, feel free to close and discard this PR.

**Pull request closed without merging**